say/say_test.go

```diff
@@ -0,0 +1 @@
+package say
```

Comment:
Do we need that file if we do not have tests for that?

Reply:
at first it asked me for it. but i will try and delete it again

Reply:
If we need to keep it we could also just write some tests ;)